pkg/machinery/client/client.go

```diff
@@ -167,7 +169,10 @@ func New(ctx context.Context, opts ...OptionFunc) (c *Client, err error) {
 
 // GetConn creates new gRPC connection.
 func (c *Client) GetConn(ctx context.Context, opts ...grpc.DialOption) (*grpc.ClientConn, error) {
```

Comment:
let's make this method private, iirc it's not being used outside of this module, and it doesn't make sense to keep it public